packages/moonstone/Scrollable/ScrollButton.js

```diff
@@ -88,14 +97,15 @@ const ScrollButton = kind({
 		className: ({direction, styler}) => styler.append(classNameMap[direction])
 	},
 
-	render: ({children, disabled, ...rest}) => {
+	render: ({css, children, disabled, ...rest}) => {
```

Comment:
Let `css` be in `rest` in peace.

Reply:
Hmm... It looks that this code works with desctructuring `css` because `css` is NOT _enumarable_. And, IMO, we don't need to define `css` prop for `ScrollButton` due to this, but I'm not sure what is the better way to implement this.

Reply:
I passed `css` variable from not the arguments of the render function but the variable imported from './ScrollButton.less' directly.

Reply:
💨